src/main/java/games/strategy/triplea/delegate/BattleCalculator.java

```diff
@@ -1345,13 +1339,12 @@ public static int getUnitPowerForSorting(final Unit current, final boolean defen
       final Collection<TerritoryEffect> territoryEffects) {
     final boolean lhtrBombers = games.strategy.triplea.Properties.getLHTR_Heavy_Bombers(data);
     final UnitAttachment ua = UnitAttachment.get(current.getType());
-    int rolls;
+    final int rolls;
     if (defending) {
       rolls = ua.getDefenseRolls(current.getOwner());
     } else {
       rolls = ua.getAttackRolls(current.getOwner());
     }
```

Comment:
Yes, but the intent is just to apply the keyword, not looking to do anything else here. I would like to keep the scope restricted to that. There are a number of reasons not to increase scope:

- it's a slippery slope, can pick up any number of further updates in nearby code, and then further updates when updating that code
- drastically increases test scope, applying final keyword via automatic IDE detection is pretty safe. Updating if statements should be done a bit more carefully.
- keep the diff clean. If I loop in a number of changes like this, then we would have a multi-hundred line diff with a number of different changes. This makes the branch harder to review, which slows it up, increases potential for merge conflict since change set is larger and longer lived, and any problem in one set of changes delays the whole thing.
- we can follow up with these types of changes in subsequent iterations, it's important to keep in mind the intent and reason for changes, and the specific goal of the current iteration.

(will get off the soap box now, but I think those are some important points to keep in mind when requesting changes)